Userland/Libraries/LibJS/Tests/test-common.js

```diff
@@ -611,4 +612,24 @@ class ExpectationError extends Error {
             duration: 0,
         };
     };
+
+    withinSameSecond = callback => {
+        let callBackTime = -1;
+        for (let tries = 0; tries < 5; tries++) {
+            const startDate = Temporal.Now.instant().epochNanoseconds;
```

Comment:
This is still a bit awkward. Let's

- Stop the reference to "date", i.e. just `start` and `end`
- Store a plain `Instant` object, not one of its internal values
- Simply compare `start.epochSeconds !== end.epochSeconds`